tests/system/action/user/test_create.py

```diff
@@ -483,33 +617,77 @@ def test_create_permission_group_A_no_permission(self) -> None:
             response.json["message"],
         )
 
-    # Rm template fields makes this test useless.
-    @pytest.mark.skip
     def test_create_permission_group_B_user_can_manage(self) -> None:
         """create group B fields with simple user.can_manage permissions"""
         self.permission_setup()
         self.set_organization_management_level(None, self.user_id)
         self.set_user_groups(self.user_id, [2])  # Admin groups of meeting/1
 
+        self.set_models(
+            {
+                "user/5": {"username": "user5"},
+                "user/6": {"username": "user6"},
+            }
+        )
+        self.set_user_groups(5, [1])
+        self.set_user_groups(6, [1])
```

Comment:
Maybe change `set_user_groups` so you can set the groups of all three users in one write request.

Reply:
I think it isn't used very often